integration/testdata/generic-attestation-policy/test-attestor-2.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kritis.grafeas.io/v1beta1
```

Comment:
Is this needed? I don't see any references to it in the PR.

Reply:
I think I'll add another test that uses it.  I can do it in a separate PR if you prefer it, in which case I'll get rid of these unused objects.  Let me know.

Reply:
Yes please, let's make sure we keep the history as understandable as possible. Thanks!

Reply:
Instead of adding another test, I changed TestKritisGAPLogic very slightly.  The policy under test now contains two attestors, to make sure that needs-only-one-attestor semantics is exercised.  And the test now uses test-attestor-2.  WDYT?

Reply:
Perfect, thank you. I like the updated version of the test. One minor optional thing: if you could please comment on top of the test the high-level of what the test is verifying, that'd be fantastic.

Reply:
Thanks, done.